.github/workflows/es-otel-integration-tests.yml

```diff
@@ -0,0 +1,23 @@
+name: Integration Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  elasticsearch-otel:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+
+    - name: Install tools
+      run: make install-ci
+
+    - name: Run elasticsearch otel integration tests
+      run: bash scripts/travis/es-integration-test.sh otel
```

Comment:
It may be worth rethinking this test. The script internally runs 3 tests with different S versions
```
run_integration_test "5.6.16"
run_integration_test "6.8.2"
run_integration_test "7.3.0"
```
They can be easily run as different jobs in the same workflow, with the same granularity of restart as today. And afair starting ES is what takes the most time, not the tests themselves, so it would be much more efficient if we ran both default and otel tests sequentially after a given ES is started.

Reply:
Good point. Sure will convert to separate jobs.

Reply:
Hi @yurishkuro 

I have updated the PR splitting each into different jobs. Thank you.